src/pages/FindID/FindID.scss

```
@@ -78,57 +80,35 @@
     }
   }
 
-  // 모달 창 스타일
-  .modal {
-    display: none;
-    position: fixed;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    background-color: rgba(0, 0, 0, 0.7);
-    z-index: 1;
+  .verifinButtonFrame {
+    display: flex;
+    flex-direction: column;
     justify-content: center;
     align-items: center;
-  }
-
-  .modal-content {
-    background-color: #fff;
-    padding: 20px;
-    border-radius: 5px;
-    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
-    position: relative;
-    max-width: 80%;
-    max-height: 80%;
-    overflow: auto;
-    text-align: center;
-  }
-
-  .close {
-    position: absolute;
-    top: 10px;
-    right: 10px;
-    font-size: 20px;
-    cursor: pointer;
-  }
-
-  // 모달 내부 스타일
-  .modal p {
-    font-size: 18px;
-    margin: 20px 0;
-  }
+    margin-top: 20px;
 
-  .modal button {
-    background-color: #007bff;
-    color: #fff;
-    padding: 10px 20px;
-    border: none;
-    cursor: pointer;
-    font-size: 16px;
-    border-radius: 5px;
-  }
+    .disabledButton {
+      width: 340px;
+      padding: 15px;
+      text-align: center;
+      border: none;
+      border-radius: 5px;
+      background-color: rgb(184, 184, 184);
+      font-weight: 700;
+      color: rgb(105, 105, 105);
+      font-size: 18px;
+    }
 
-  .modal button:hover {
-    background-color: #0056b3;
+    .findIDButton {
+      width: 340px;
+      padding: 15px;
+      text-align: center;
+      border: none;
+      border-radius: 5px;
+      background-color: rgb(47, 130, 255);
+      font-weight: 700;
+      color: white;
+      font-size: 18px;
+    }
```

color 관련된 내용만 nesting하여 처리하면 될 것 같습니다.
```suggestion
    .findIDButton {
      width: 340px;
      padding: 15px;
      text-align: center;
      border: none;
      border-radius: 5px;
      background-color: rgb(47, 130, 255);
      font-weight: 700;
      color: white;
      font-size: 18px;
      
      &.disabled {
        background-color: rgb(184, 184, 184);
        color: rgb(105, 105, 105);
      }
    }
```